conf/janus.transport.mqtt.cfg.sample

```diff
@@ -0,0 +1,36 @@
+; Configuration of the MQTT additional transport for the Janus API.
+[general]
+enable = no                   ; Whether the support must be enabled
+json = indented               ; Whether the JSON messages should be indented (default),
+                              ; plain (no indentation) or compact (no indentation and no spaces)
+
+url = tcp://localhost:1883    ; The connection URL of the MQTT broker
+;client_id = guest            ; Client identifier
+;username = guest             ; Username to use to authenticate, if needed
+;password = guest             ; Password to use to authenticate, if needed
+
+[connect]
+;keep_alive_interval = 20     ; Keep connection for N seconds
+;cleansession = 0             ; Clean session flag
+
+[disconnect]
+;timeout = 100                ; Seconds to wait before destroying client
+
+[subscribe]
+topic = to-janus              ; Topic for incoming messages
+;qos = 1                      ; QoS for incoming messages
+
+[publish]
+topic = from-janus            ; Topic for outgoing messages
+;qos = 1                      ; QoS for outgoing messages
+
+[admin.subscribe]
```

Comment:
I'd prefer the prefixed approach there, yep, mostly because it makes it easier for who configures the plugin to know what belongs where (and because it's consistent with the way you configure other transports).